app-dicts/aspell-en/aspell-en-2018.04.16.0.ebuild

```diff
@@ -0,0 +1,15 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
This copyright header is old. Please use repoman to commit to fix it for you automatically.

Reply:
Done.